modules/backend/formwidgets/Repeater.php

```diff
@@ -366,4 +374,13 @@ public function getGroupTitle($groupCode)
     {
         return array_get($this->groupDefinitions, $groupCode.'.name');
     }
+
+    protected function processForMinItems()
+    {
+        if($this->minItems === null) return;
+
+        if ($this->indexCount <= $this->minItems) {
```

Comment:
Why is this a `<=`? instead of just a `<` sign?

Reply:
The minimum requirement items will be created from the developer. 

`<=?` instead of just a `<` becasue indexCount begins from 1.